cassovary-core/src/main/scala/com/twitter/cassovary/graph/MemoryMappedDirectedGraph.scala

```diff
@@ -51,48 +70,63 @@ class MemoryMappedDirectedGraph(file: File) extends DirectedGraph[Node] {
   private class MemoryMappedDirectedNode(override val id: Int) extends Node {
     val nodeOutboundOffset = outboundOffset(id)
     val nodeInboundOffset = inboundOffset(id)
-    override def outboundNodes(): Seq[Int] = new IndexedSeq[Int] {
+    def outboundNodes(): Seq[Int] = new IndexedSeq[Int] {
       val length: Int = outDegree(id)
       def apply(i: Int): Int =  data.getInt(nodeOutboundOffset + 4L * i)
-      override def toString(): String = "Memory Mapped IndexedSeq " + mkString("[", ",", "]")
     }
-    override def inboundNodes(): Seq[Int] = new IndexedSeq[Int] {
+    def inboundNodes(): Seq[Int] = new IndexedSeq[Int] {
       val length: Int = inDegree(id)
       def apply(i: Int): Int =  data.getInt(nodeInboundOffset + 4L * i)
-      override def toString(): String = "Memory Mapped IndexedSeq " + mkString("[", ",", "]")
     }
   }
 
-  override def getNodeById(id: Int): Option[Node] =
+  def getNodeById(id: Int): Option[Node] =
     if(0 <= id && id < nodeCount) {
       Some(new MemoryMappedDirectedNode(id))
     } else {
       None
     }
 
-  override def iterator: Iterator[Node] = (0 to nodeCount).iterator flatMap (i => getNodeById(i))
+  def iterator: Iterator[Node] = (0 to nodeCount).iterator flatMap (i => getNodeById(i))
 
-  override def edgeCount: Long = (outboundOffset(nodeCount) - outboundOffset(0)) / 4
+  lazy val edgeCount: Long = outboundOffset(nodeCount) - outboundOffset(0)
 
-  override lazy val maxNodeId: Int = nodeCount - 1
-  // Uncomment in future if maxNodeId becomes a method:
-  // override def maxNodeId = nodeCount - 1
+  override lazy val maxNodeId = nodeCount - 1
 
-  override val storedGraphDir = StoredGraphDir.BothInOut
```

Comment:
Same as above.  I think having the override is slightly clearer, but I couldn't find an official style statement anywhere.  There's a stack overflow thread somewhere where people debate the issue, with no clear conclusion.